ilastik/applets/layerViewer/layerViewerGui.py

```diff
@@ -310,6 +356,72 @@ def _create_grayscale_layer_from_slot(cls, slot, n_channels):
         return layer
 
     @classmethod
+    def createGrayscaleLayer(cls, slot):
```

Comment:
Do we really need these functions? There is already an addGrayscaleLayer in the viewer, what's the advantage of this function? I don't mind adding it if you think it makes developer life easier, just want to avoid too many functions doing almost the same thing.